tachyon/crypto/commitments/kzg/shplonk.h

```diff
@@ -40,6 +41,7 @@ class SHPlonk final : public UnivariatePolynomialCommitmentScheme<
       SHPlonk<Curve, MaxDegree, Commitment>>;
   using G1Point = typename Curve::G1Curve::AffinePoint;
   using G2Point = typename Curve::G2Curve::AffinePoint;
+  using G2Prepared = typename Curve::G2Prepared;
```

Comment:
I thought you modified the pairing logic to be able to use only w/ `G1Point` and `G2 Point` w/o `G2Prepared` in this [commit](https://github.com/kroma-network/tachyon/pull/222/commits/779eb27b82cc4f78d26ff2289d5653606670300d#diff-d51496b909572d7b364c5dedca54853dba6ac3af77c865bc8ce5c511ebb569b4).
Is there any reason to use it again?

Reply:
`G2Prepared` looks like an expensive operation and in our case, the values are fixed so we computed these values in advance and we uses them.